closure/goog/demos/keyhandler.html

```diff
@@ -98,6 +98,20 @@
   }
 
   switch (e.keyCode) {
+
+  // Add the keycodes for the number-pad keys
+
+	case goog.events.KeyCodes.NUM_ZERO:
```

Comment:
It'd be easier if you just edit it. Just indent the cases properly with spaces and not tabs.